models/phonedetails.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class PhoneDetails extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+  PhoneDetails.init({
+    phoneId: DataTypes.STRING,
+    namespaceId: DataTypes.STRING,
+    name: DataTypes.STRING,
+    capacityAvailable: DataTypes.ARRAY(DataTypes.STRING),
+    capacity: DataTypes.STRING,
+    priceRegular: DataTypes.INTEGER,
+    priceDiscount: DataTypes.INTEGER,
+    colorsAvailable: DataTypes.ARRAY(DataTypes.STRING),
+    color: DataTypes.STRING,
+    images: DataTypes.ARRAY(DataTypes.STRING),
+    description: DataTypes.ARRAY(DataTypes.JSON),
```

Comment:
how about creating specific object, instead of just passing JSON? Alternatively you could create a separate model for `description`
```
{
  title: DataTypes.STRING
  text: DataTypes.STRING
}
```